docs/Makefile

```diff
@@ -18,7 +18,7 @@ clean:
 
 .PHONY: html
 html:
-	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html $(SPHINXOPTS)
+	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html $(SPHINXOPTS) -j 'auto'
```

Comment:
Oops, accidentally removed in https://github.com/alteryx/evalml/pull/1959 so adding back in